doc/modules/compose.rst

```diff
@@ -6,13 +6,14 @@ Pipelines and composite estimators
 ==================================
 
 Transformers are usually combined with classifiers, regressors or other
-estimators to build a composite estimator.  The most common tool is a
-:ref:`Pipeline <pipeline>`. Pipeline is often used in combination with
-:ref:`FeatureUnion <feature_union>` which concatenates the output of
-transformers into a composite feature space.  :ref:`TransformedTargetRegressor
-<transformed_target_regressor>` deals with transforming the :term:`target`
-(i.e. log-transform :term:`y`). In contrast, Pipelines only transform the
-observed data (:term:`X`).
+:term:`predictors` to build a composite estimator. The most common tool is a
```

Comment:
I like the new paragraph. It is easier to read and is still correct. Getting this right is not easy, at least I had to go and read the definitions of estimator, predictor, etc and then think to figure out if it was correct or not!

I find the start of the first sentence hard to read. What do you think of starting with the end "To build a composite estimator ...."?

Another question: would it be correct to say "... combined with classifiers, regressors, other predictors or estimators to build a composite estimator"? The reason I'm asking is that if I want to make a composite estimator that is a transformer, then I don't need a predictor as the last step no? The model I'm starting to form in my head is that the type of the composite estimator is equal to the type of the final step in the pipeline. Is that useful as a mental model?

Reply:
@betatim Thanks for your review. Your thoughts helped me to clarify how to communicate this. I've changed how to start the first sentence and found some words to express, that Pipeline would also combine two transformers, and what the type of the pipeline then would be.

This is the result:
```
To build a composite estimator, transformers are usually combined with other
transformers or with :term:`predictors` (such as classifiers or regressors).
The most common tool used for this is a :ref:`Pipeline <pipeline>`. Pipelines
without a final predictor will fit and transform the observed data (:term:`X`)
and return it. In contrast, pipelines with a final predictor will fit the
predictor with the transformed data from the previous steps and can then also
be used for predictions. The methods to be called on the pipeline are the same
methods that the last contained estimator provides.
```

What do you think?

Reply:
I like it!